### DISCUSSION TITLE
How I add the $loop->index values in ID column

### DISCUSSION THREAD

**Discussion by mani1430** [Power-Components/livewire-powergrid#386] [Q&A]
1. How I add the $loop->index values in ID column. I don't want to use database table primary id values in the table row number.

2. I need to use encrypted URL , So please help me to how to encrypt the edit route parameter.

please help me.

like =>  Button::add('edit')
               ->target('')
               ->caption('<i class="fa-solid fa-pen-to-square"></i>')
               ->class(' cursor-pointer text-sky-600 px-3 py-2.5 m-1 rounded text-sm')
               ->route('customer.edit', ['url' =>  [ 'id' , Crypt::encrypt($id);]]),

how I don't have any idea , please help me.

3. Need to place Actions buttons near to id columns in the table.

![Screenshot (54)](https://user-images.githubusercontent.com/90332512/162664760-da30d1b7-c596-4f77-ab65-fa8a3797153f.png)

**Comment by dansysanalyst** [comment]
Hi @mani1430,

Please use the **Discussions** for questions about "how to...", "how do I?".

We prefer to keep "Issues" exclusively to track Bugs and other problems.

Thanks

**Reply by mani1430** [reply]
help me please , how do I ?
